Formula/spotbugs.rb

```diff
@@ -0,0 +1,21 @@
+class Spotbugs < Formula
+  desc "SpotBugs is FindBugs' successor. A tool for static analysis to look for bugs in Java code."
+  homepage "https://spotbugs.github.io/"
+  url "https://github.com/spotbugs/spotbugs/archive/3.1.0.tar.gz"
+  sha256 "ac6169a551212756a05dd9c066ef9212653d8a99b754ef4dade6a7008f8ad781"
+  head "https://github.com/spotbugs/spotbugs.git"
+
+  depends_on "gradle" => :build
+  depends_on :java => "1.8+"
+
+  def install
+    system "gradle build"
+    system "gradle installDist"
+    libexec.install Dir["spotbugs/build/install/spotbugs/*"]
+    bin.install_symlink "#{libexec}/bin/spotbugs"
```

Comment:
any reason none of the others in libexec/bin?

Reply:
`spotbugs` is a wrapper that sets some other variables, etc. they may work depending on system configuration, but `spotbugs` is the best internally curated binary